demisto_sdk/commands/validate/tests/RP_validatiors_test.py

```diff
@@ -55,3 +58,49 @@ def test_ExpirationFieldIsNumericValidator_is_valid(
             for result, expected_msg in zip(results, expected_msgs)
         ]
     )
+
+
+@pytest.mark.parametrize(
```

Comment:
as you might have heard - avoid using pytest.mark.parametrize
instead, split into different test methods, each one with more specific naming and docstring based on the test case.
Another option: just call the is_valid() multiple times in the same method. Still, I'd split into "valid" and "invalid" test methods.

Generally, this decision was made collectively in the team a few months ago, and the goal is for better readability and maintenance for the tests.
there are some cases where it is ok to use parametrize, e.g. - in very very simple tests with up to 2 arguments, that shouldn't be modified with more test-cases over the time. When you think this is the case, consult with Eyal/me if you want to use it.